pkg/resources/apiserver/deployment.go

```diff
@@ -498,7 +510,7 @@ func getApiserverOverrideFlags(data *resources.TemplateData) (kubermaticv1.APISe
 	return settings, nil
 }
 
-func getVolumeMounts(isKonnectivityEnabled, isEncryptionEnabled bool, isAuditWebhookEnabled bool) []corev1.VolumeMount {
+func getVolumeMounts(data *resources.TemplateData, isKonnectivityEnabled, isEncryptionEnabled bool, isAuditWebhookEnabled bool) []corev1.VolumeMount {
```

Comment:
Since `isKonnectivityEnabled` is derived from `data` in the call to `getVolumeMounts`, we should probably remove the superfluous parameter here.